common/hashing.h

```diff
@@ -59,8 +59,7 @@ class HashCode : public Printable<HashCode> {
   // The index will be in the range [0, `size`). The `size` must be a power of
   // two, and `N` must be in the range [1, 32].
   template <int N>
-  constexpr auto ExtractIndexAndTag(ssize_t size)
-      -> std::pair<ssize_t, uint32_t>;
+  constexpr auto ExtractIndexAndTag() -> std::pair<ssize_t, uint32_t>;
```

Comment:
Update doc comment to reflect this change

Reply:
Done, thanks!